### ISSUE TITLE
Nitropy fails to connect with the Nitrokey Pro on Windows 10

### ISSUE THREAD

**Issue by Niklap97** [Nitrokey/pynitrokey#98]
Running "nitropy pro"- commands with a Pro device causes an error:
![nitropy_failed_to_execute_nitropy_git](https://user-images.githubusercontent.com/64582081/139439271-7667fa9c-6e9f-4f57-b245-883c4600178d.PNG)

Environment:
- Nitrokey tool for Nitrokey FIDO2, Nitrokey Start & NetHSM 0.4.4
- Windows 10

**Comment by Niklas-Nitrokey** [comment]
The same error occurs with the 0.4.5 version

**Comment by szszszsz** [comment]
@daringer Hi! Can you take a moment to look at this issue? It seems to be about Windows packaging, but I might be wrong.

**Comment by daringer** [comment]
yes this looks like the header files are not packaged together into the windows package. But libnitrokey is also needed, is there a libnitrokey available on this system (looks like it is set via env). So assuming the lib is there and all the other bits are working, best  case `wine-build/nitropy.spec.tmpl` just needs an addition to `datas` containing the `nk_headers` dir...

**Comment by robin-nitrokey** [comment]
@Niklap97 Do you remember whether you used the binary or the installer?  Can you please try again with the binary from the latest release?

**Comment by Niklap97** [comment]
@robin-nitrokey I used the binary. 
Its still the same for the latest release.
